### ISSUE TITLE
Part3A

### ISSUE THREAD

**Issue by krittichier** [csreid/CS580-P5#6]

**Comment by csreid** [comment]
@kjrittichier This is just writing a bunch of `0`s to `coded.txt` for me.

**Comment by krittichier** [comment]
@csreid: Weird. Did you generate this one (https://github.com/csreid/CS580-P5/blob/master/coded.txt) because I don't remember uploading this and I don't see it on the Part3A branch?

**Comment by csreid** [comment]
Yep, I did, with the Python script in `/scripts`. Just to get myself
something to work with

**Comment by krittichier** [comment]
@csreid: Okay, that's fair!! 
What input is giving you all zeros?

**Comment by csreid** [comment]
Just that normal "aaaaabbbbcccddddeeefffffff" test file.

I'm planning to head to the library around 4:30ish; we can check then. I've been having weird compiler issues all semester so it might be something simple

**Comment by krittichier** [comment]
Weird.
4:30 sounds good. I'm just chilling on the second floor. We can move somewhere else when you get here.
